Resources/Prototypes/Entities/Mobs/Species/shadowkin.yml

```
@@ -207,7 +208,7 @@
       animation: WeaponArcClaw
       damage:
         types:
-          Slash: 5
+          Slash: 8
```

```suggestion
          Slash: 5
```
This is an incredibly bad idea because of how you can accumulate melee damage modifiers in EE code. Increasing the base damage to 8 means that they can actually get their unarmed melee all the way up to 18.75 if they stack every currently available modifier for it.